### PR TITLE
Document targetPath in Helm Releases guide

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]

File: docs/guides/helmreleases.md [hunk 1]
```diff
@@ -106,18 +106,33 @@ spec:
...
 
 The definition of the listed keys is as follows:
 
 - `kind`: Kind of the values referent (`ConfigMap` or `Secret`).
 - `name`: Name of the values referent, in the same namespace as the
   `HelmRelease`.
-- `valuesKey` _(Optional)_: The key in the referent the values can be
-  found at. Defaults to `values.yaml` when ommitted.
+- `valuesKey` _(Optional)_: The data key where the values.yaml or a
+   specific value can be found at. Defaults to `values.yaml` when
```
Comment (squaremo) [inline docs/guides/helmreleases.md:122]:
```suggestion
   specific value can be found. Defaults to `values.yaml` when
```

File: docs/guides/helmreleases.md [hunk 2]
```diff
@@ -106,18 +106,33 @@ spec:
...
+   omitted.
+- `targetPath` _(Optional)_: The YAML dot notation path the value
+   should be merged at. When set, the `valuesKey` is expected to be
```
Comment (squaremo) [inline docs/guides/helmreleases.md:125]:
```suggestion
   should be merged. When set, the `valuesKey` is expected to be
```

File: docs/guides/helmreleases.md [hunk 3]
```diff
@@ -106,18 +106,33 @@ spec:
...
 
 - `kind`: Kind of the values referent (`ConfigMap` or `Secret`).
 - `name`: Name of the values referent, in the same namespace as the
   `HelmRelease`.
-- `valuesKey` _(Optional)_: The key in the referent the values can be
-  found at. Defaults to `values.yaml` when ommitted.
+- `valuesKey` _(Optional)_: The data key where the values.yaml or a
+   specific value can be found at. Defaults to `values.yaml` when
+   omitted.
+- `targetPath` _(Optional)_: The YAML dot notation path the value
```
Comment (squaremo) [inline docs/guides/helmreleases.md:124]:
```suggestion
- `targetPath` _(Optional)_: The YAML dot notation path at which the value
```

**Review by squaremo — APPROVED** [review]
Reads well :+1:. I shuffled a couple of sentences around to avoid a dangling preposition. I think examples would go a long way for the `targetPath` thing.

**Pull request merged**